tests/test_commands.py

```diff
@@ -330,6 +334,24 @@ def test_asyncio_enabled_false(self):
         self.assertNotIn("Using reactor: twisted.internet.asyncioreactor.AsyncioSelectorReactor", log)
 
 
+class WindowsRunSpiderCommandTest(RunSpiderCommandTest):
+
+    spider_filename = 'myspider.pyw'
+
+    def setUp(self):
+        if platform != 'win32':
+            raise unittest.SkipTest(".pyw file extension only supported in Windows")
```

Comment:
Do all tests fail in Linux? I think we should try to run as many of them as possible also in Linux, at least until we have Windows support in the CI.

Reply:
I checked it now 6 of them are failing maybe I should run the rest for linux too.